tests/cypress/config/verify_YAML_stability/a.yaml

```diff
@@ -0,0 +1,129 @@
+apiVersion: policy.open-cluster-management.io/v1
```

Comment:
@kkaarreell Any specific reason to have this file name as `a.yaml` ?

Reply:
Oh!. That should not be there.. pushing an update.